test/system/VectorListTest.ooc

```diff
@@ -137,34 +136,24 @@ VectorListTest: class extends Fixture {
 			(list, slice, sliceInto) free()
 		})
 		this add("VectorList apply", func {
-			list := VectorList<Int> new()
-			list add(0)
-			list add(1)
-			list add(2)
-			c := 0
-			list apply(|value|
-				expect(value, is equal to(c))
-				c += 1)
-			list free()
+			This staticlist = VectorList<Int> new()
+			This staticlist add(0) . add(1) . add(2)
+			This staticlist apply(This _applyTester)
+			This staticlist free()
 		})
 		this add("VectorList modify", func {
 			list := VectorList<Int> new()
-			list add(0)
-			list add(1)
-			list add(2)
+			list add(0) . add(1) . add(2)
 
 			list modify(|value| value += 1)
```

Comment:
The closure passed to `modify` somehow doesn't leak, in contrast to the argument to `apply`...